### PR TITLE
Update migration-tidb-faq.md

### PR DESCRIPTION
**Pull request by lance6716**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lichunzhu — COMMENTED** [review]
/lgtm

**Review by lichunzhu — APPROVED** [review]
LGTM

File: faq/migration-tidb-faq.md [hunk 1]
```diff
@@ -50,9 +50,14 @@ DB2、Oracle 到 TiDB 数据迁移（增量+全量），通常做法有：
...
-### Dumpling 导出大表时引发上游数据库报错“磁盘空间不足”
+### Dumpling 导出时引发上游数据库 OOM 或报错“磁盘空间不足”
 
-该问题是由于数据库主键分布不均匀，Dumpling 划分导出子范围时出现过大的子范围引起的。请尝试分配更大的磁盘空间，或者联系 [AskTUG 社区专家](https://asktug.com/) 获取实验版本的 Dumpling。
+该问题可能有如下原因：
+
+- 数据库主键分布不均匀，例如启用了 [SHARD_ROW_ID_BITS](/shard-row-id-bits.md)
+- 上游数据库为 TiDB，导出表是分区表
+
+在上述情况下，Dumpling 划分导出子范围时，会划分出过大的子范围，从而向上游发送结果过大的查询。请联系 [AskTUG 社区专家](https://asktug.com/) 获取实验版本的 Dumpling。
```
Comment (TomShawn) [inline faq/migration-tidb-faq.md:60]: Any alternative channel for AskTUG in the English version? Slack?

Comment (lance6716) [reply faq/migration-tidb-faq.md:60]: Slack is OK

Comment (TomShawn) [reply faq/migration-tidb-faq.md:60]: Do you have the link for Slack?

Comment (lance6716) [reply faq/migration-tidb-faq.md:60]: https://slack.tidb.io/invite?team=tidb-community&channel=sig-migrate&ref=github_sig

Comment (TomShawn) [inline faq/migration-tidb-faq.md:60]: Rest LGTM
```suggestion
在上述情况下，Dumpling 划分导出子范围时，会划分出过大的子范围，从而向上游发送结果过大的查询。请联系 [AskTUG 社区专家](https://asktug.com/)获取实验版本的 Dumpling。
```

**Pull request merged**